bluemix-settings.js

```diff
@@ -18,8 +18,9 @@ var path = require("path");
 var util = require("util");
 var fs = require("fs");
 
-var cfenv = require("cfenv");
-var appEnv = cfenv.getAppEnv();
+const IBMCloudEnv = require('ibm-cloud-env');
+IBMCloudEnv.init('/server/config/mappings.json');
+var cloudantUrl = IBMCloudEnv.getString('cloudant_url');
```

Comment:
Can this be a `const`?

Reply:
I'm not very proficient with Node.js, so I'm not sure. Maybe @youming-lin or @gee4vee can comment on that.

Reply:
I changed `var cloudantUrl` to `const cloudantUrl`